apps/mobile/src/screens/CreatePasscode.tsx

```diff
@@ -0,0 +1,15 @@
+import { CreatePasscode } from '@walless/app';
```

Comment:
component name should be `CreatePasscodeFeature` which self-explains it is a shared feature.